### PR TITLE
Remove celery tasktree and use a regular celery chain/group instead.

### PR DESCRIPTION
**Pull request by EnTeQuAk**

### CODE REVIEW

**Comment by muffinresearch** [comment]
more-red-than-green-award-sm

File: requirements/prod.txt [hunk 1]
```diff
@@ -47,11 +47,9 @@ boto==2.20.0 \
 # cef is required by heka-py-cef
 cef==0.5 \
     --hash=sha256:2d1cb558391673df5109a3fe5e84fde6160edc8b241b3eb19ab3d3abe9b8df31
-celery==3.1.20 \
-    --hash=sha256:3071b71ef8c43178ace8435002b11f2ff06db7690f07d960540eab7f4183ddf7 \
-    --hash=sha256:d02f191c3d92a851c9d2028e91baf2a0f2734cd3b659743d3624011d1ef7a6d5
-celery-tasktree==0.3.2 \
-    --hash=sha256:06be6b5f63bc2b5b252a9f210513d9b81e6813b099a554cb78e8754a1708774e
+celery==3.1.23 \
```
Comment (diox) [inline requirements/prod.txt]: Can you also bump kombu, billiard, amqp ? i.e. the whole celery stack ?

**Comment by diox** [comment]
flake8 issue :)

**Comment by diox** [comment]
r+ (we'll want to launch the reindex command on dev once this lands, to test it out)

**Pull request merged**